lighty-modules/lighty-aaa-aggregator/lighty-aaa/pom.xml

```diff
@@ -72,6 +72,10 @@
             <artifactId>mockito-inline</artifactId>
             <scope>test</scope>
         </dependency>
+      <dependency>
```

Comment:
What's the purpose of this change? I need to recheck our dependencies...

Reply:
as stated in c5936074d63b807ccd8852975ed7b4ba1e85476c This import is required for AuthenticationManager.java, HttpBasicAuth.java and TokenAuthenticators.java used in AAALightyShiroProvider